opencensus/proto/agent/trace/v1/trace_service.proto

```diff
@@ -33,6 +33,8 @@ service TraceService {
   // After initialization, this RPC must be kept alive for the
   // entire life of the application. The agent pushes configs
   // down to applications via a stream of requests.
+  // Logically the requests on updating config are coming from agent,
```

Comment:
the requests to update configurations come from the agent
and that's why the message name is suffixed with "Request"